src/Orc.NuGetExplorer.Xaml/ViewModels/ExplorerViewModel.cs

```diff
@@ -131,9 +135,26 @@ private void InitializePages()
             _pageSetup.Values.ForEach(page =>
                 Pages.Add(_typeFactory.CreateInstanceWithParametersAndAutoCompletion<ExplorerPage>(page)));
 
+            foreach (var page in Pages)
+            {
+                page.PropertyChanged += OnPageActiveChanged;
```

Comment:
I recommend moving events subscription into `Initialize()` method
And don't forget to unsubscribe in `Close()`

Reply:
I think i simplify to moving all content of this method to InitializeAsync(), it doesn't create any value